src/components/form/wizard/index.tsx

```diff
@@ -79,10 +81,22 @@ const Wizard = ({ children, show = true, steps_names, title, enable_next_button
 
     if (!show_wizard) return <></>
 
+    const showDialogBox = () => {
+        setCloseWizard(true)
+    }
+
     return (
         <>
             <Modal>
-                <Header title={title} setShowWizard={setShowWizard} />
+                {close_wizard && (
+                    <DialogModelBox
+                        toggle={() => setCloseWizard(false)}
+                        is_open={close_wizard}
```

Comment:
if `is_open` props is `false` I assume it will already hide the dialog box, do we still need the 

```
{close_wizard && (
```

on the upper scope?